tex/generic/pgf/utilities/pgfcalendar.code.tex

```diff
@@ -797,5 +802,207 @@
   \fi}%
 }
 
+%
+% more tests
+%
+\pgfqkeys{/pgf/calendar}{
+  Jan/.code={\ifnum\pgfcalendarifdatemonth=1\pgfcalendarmatchestrue\fi},Jan/.value forbidden,
+  Feb/.code={\ifnum\pgfcalendarifdatemonth=2\pgfcalendarmatchestrue\fi},Feb/.value forbidden,
+  Mar/.code={\ifnum\pgfcalendarifdatemonth=3\pgfcalendarmatchestrue\fi},Mar/.value forbidden,
+  Apr/.code={\ifnum\pgfcalendarifdatemonth=4\pgfcalendarmatchestrue\fi},Apr/.value forbidden,
+  May/.code={\ifnum\pgfcalendarifdatemonth=5\pgfcalendarmatchestrue\fi},May/.value forbidden,
+  Jun/.code={\ifnum\pgfcalendarifdatemonth=6\pgfcalendarmatchestrue\fi},Jun/.value forbidden,
+  Jul/.code={\ifnum\pgfcalendarifdatemonth=7\pgfcalendarmatchestrue\fi},Jul/.value forbidden,
+  Aug/.code={\ifnum\pgfcalendarifdatemonth=8\pgfcalendarmatchestrue\fi},Aug/.value forbidden,
+  Sep/.code={\ifnum\pgfcalendarifdatemonth=9\pgfcalendarmatchestrue\fi},Sep/.value forbidden,
+  Oct/.code={\ifnum\pgfcalendarifdatemonth=10\pgfcalendarmatchestrue\fi},Oct/.value forbidden,
+  Nov/.code={\ifnum\pgfcalendarifdatemonth=11\pgfcalendarmatchestrue\fi},Nov/.value forbidden,
+  Dec/.code={\ifnum\pgfcalendarifdatemonth=12\pgfcalendarmatchestrue\fi},Dec/.value forbidden,
```

Comment:
```suggestion
  Jan/.code={\ifnum\pgfcalendarifdatemonth=1\relax\expandafter\pgfcalendarmatchestrue\fi},Jan/.value forbidden,
  Feb/.code={\ifnum\pgfcalendarifdatemonth=2\relax\expandafter\pgfcalendarmatchestrue\fi},Feb/.value forbidden,
  Mar/.code={\ifnum\pgfcalendarifdatemonth=3\relax\expandafter\pgfcalendarmatchestrue\fi},Mar/.value forbidden,
  Apr/.code={\ifnum\pgfcalendarifdatemonth=4\relax\expandafter\pgfcalendarmatchestrue\fi},Apr/.value forbidden,
  May/.code={\ifnum\pgfcalendarifdatemonth=5\relax\expandafter\pgfcalendarmatchestrue\fi},May/.value forbidden,
  Jun/.code={\ifnum\pgfcalendarifdatemonth=6\relax\expandafter\pgfcalendarmatchestrue\fi},Jun/.value forbidden,
  Jul/.code={\ifnum\pgfcalendarifdatemonth=7\relax\expandafter\pgfcalendarmatchestrue\fi},Jul/.value forbidden,
  Aug/.code={\ifnum\pgfcalendarifdatemonth=8\relax\expandafter\pgfcalendarmatchestrue\fi},Aug/.value forbidden,
  Sep/.code={\ifnum\pgfcalendarifdatemonth=9\relax\expandafter\pgfcalendarmatchestrue\fi},Sep/.value forbidden,
  Oct/.code={\ifnum\pgfcalendarifdatemonth=10\relax\expandafter\pgfcalendarmatchestrue\fi},Oct/.value forbidden,
  Nov/.code={\ifnum\pgfcalendarifdatemonth=11\relax\expandafter\pgfcalendarmatchestrue\fi},Nov/.value forbidden,
  Dec/.code={\ifnum\pgfcalendarifdatemonth=12\relax\expandafter\pgfcalendarmatchestrue\fi},Dec/.value forbidden,
```

Reply:
Can we do this consistently, too? Not one of the other `/pgf/calendar` tests use `\expandafter` in this matter.